packages/block-library/src/block/edit.js

```diff
@@ -250,7 +260,9 @@ export default function ReusableBlockEdit( {
 	// Apply the initial overrides from the pattern block to the inner blocks.
 	useEffect( () => {
 		defaultContent.current = {};
-		const editingMode = getBlockEditingMode( patternClientId );
+		const { getBlockEditingMode } = registry.select( blockEditorStore );
```

Comment:
Why use `registry.select` here over the alternatives?

It might be worth explaining it in a comment.

Reply:
It's basically the same thing as the alternative (using `getBlockEditingMode` from `useSelect`). I don't mind switching over just to make it easier to understand though!

Reply:
No real preference for me, just my curiosity.